frontend/src/locales/de.json

```diff
@@ -221,6 +222,10 @@
       "update": "Aktualisieren"
     },
     "changeLanguage": "Sprache ändern",
+    "empty": {
+      "list": "Leer | Noch keine {entities}",
+      "result": "Kein Resultat | Keine Resultate | Keine Resultate für {search} | Keine {search} gefunden"
+    },
```

Comment:
Made a generic way to display empty states

Reply:
How can we use this four-way pluralization?

Reply:
Hm 🤔 I think that I have misused this feature. It handles several issues in one key.

Reply:
Would you remove that again?

Reply:
I think so, as long as we can't use it. As far as I understand, only languages with other pluralization rules, such as russian, can use more than 3 pluralized entries. If you really need both "Keine Resultate für {search}" and "Keine {search} gefunden", I'd propose you create two different translation keys.